crate/kmip/src/kmip/data_to_encrypt.rs

```diff
@@ -27,6 +27,7 @@ pub struct DataToEncrypt {
 
 impl DataToEncrypt {
     /// Serialize the data to encrypt to bytes
+    #[must_use]
     pub fn to_bytes(&self) -> Vec<u8> {
         let mut bytes = Vec::new();
```

Comment:
Use `with_capacity()` instead.

Reply:
done in https://github.com/Cosmian/kms/pull/16/commits/2926f421f66d1b589c1e33262ab0dd1d85a375aa